osxkeychain/osxkeychain.go

```diff
@@ -176,7 +176,7 @@ func FindAndValidateIdentity(identityLabel string) (*IdentityWithRefModel, error
 //             you can use the ReleaseIdentityWithRefList method to do that
 func FindIdentity(identityLabel string) ([]IdentityWithRefModel, error) {
 
-	queryDict := C.CFDictionaryCreateMutable(nil, 0, nil, nil)
+	queryDict := C.CFDictionaryCreateMutable(C.kCFAllocatorDefault, 0, nil, nil)
```

Comment:
Any reference/docs why `C.kCFAllocatorDefault`?

Reply:
So the change was based on https://github.com/keybase/go-keychain/pull/31 where it was explained:

> [golang/go@94076fe](https://github.com/golang/go/commit/94076feef) made C type checking stricter.
> This caused problems where we were passing in nil for CFAllocatorRef, as Go 1.11
> will require those to be 0 instead.
> 
> Use the typed constant kCFAllocatorDefault to avoid this problem.
> 
> Also add a .travis.yml config and test on various Go versions, including master.
> 
> This PR is based on code from #30 by
> @kevinburkeomg.